common/source/docs/common-positioning-landing-page.rst

```diff
@@ -69,7 +69,8 @@ These GPS can incorporate real time kinematic data, either internally generated
     Swift Navigation's Piksi Multi RTK GPS Receiver <common-piksi-multi-rtk-receiver>
     Septentrio AsteRx-mUAS RTK GPS <common-gps-septentrio>
     Synerx MDU-2000 RTK + LTE GPS <common-synerex-mdu-2000>
-    Trimble BD930 RTK GPS <common-gps-trimble>
+    Trimble BD930 RTK GNSS <common-gps-trimble-bd930>
+    Trimble PX-1 RTX GNSS+INS <common-gps-trimble-px1>
```

Comment:
I put this in the RTK header because it's similar to RTK and don't think a single GPS is deserving of its own category (yet).